modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js

```diff
@@ -343,12 +343,12 @@
 
 					var img = document.createElement('img');
 					img.src = imagePath;
-					img.alt =
-						r[2] === undefined
-							? options && options.defaultImageText
-								? options.defaultImageText
-								: ''
-							: r[2].replace(/~(.)/g, '$1');
+					if(r[2] != undefined){
+						img.alt = r[2].replace(/~(.)/g, '$1');
+					}
+					else if( options && options.defaultImageText) {
+						img.alt = opstions.defaultImageText
```

Comment:
opstions == options

Spacing should be consistent with the rest of the code (there's a blank just after the open paren)

Reply:
Thanks for reviewing, I'll fix it based on your comments and send you a new pull request.